tutorials/azureml-in-a-day/azureml-in-a-day.ipynb

```diff
@@ -0,0 +1,765 @@
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "As you can see in this script, once the model is trained, the model file is saved and registered to the workspace. Now you can use the registered model in inferencing endpoints.\n",
+    "\n",
+    "## Configure the Command\n",
+    "Now that you have a script that can perform the desired tasks, You'll use the general purpose **command** that can run command line actions. This command line action can be directly calling system commands or running a script. \n",
+    "\n",
+    "* Here, you'll use input data, split ratio, learning rate and registered model name as input variables. Note that \n",
+    "* You will use the compute created earlier to run this command.\n",
+    "* You will use the environment created earlier - note that you can use the `@latest` notation to indicate the latest version of the environment when the command is run.\n",
+    "* You will configure some metadata like display name, experiment name etc. An experiment is a container for all the iterations one does on a certain project. All the jobs submitted under the same experiment name would be listed next to each other in Azure ML studio.\n",
+    "* You will confogure the command line action itself - `python main.py` in this case. The inputs/outputs are accessible in the command via the `${{ ... }}` notation."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from azure.ai.ml import command\n",
+    "from azure.ai.ml import Input\n",
+    "\n",
+    "registered_model_name = \"credit_defaults_model\"\n",
+    "\n",
+    "job = command(    \n",
+    "    inputs=dict(\n",
+    "        data=Input(type=\"uri_file\", path=\"https://archive.ics.uci.edu/ml/machine-learning-databases/00350/default%20of%20credit%20card%20clients.xls\"),\n",
+    "        test_train_ratio=0.2,\n",
+    "        learning_rate=0.25,\n",
+    "        registered_model_name=registered_model_name\n",
+    "        ),\n",
+    "    code=\"./src/\",#location of source code\n",
+    "    command=\"python main.py --data ${{inputs.data}} --test_train_ratio ${{inputs.test_train_ratio}} --learning_rate ${{inputs.learning_rate}} --registered_model_name ${{inputs.registered_model_name}}\",\n",
+    "    environment=\"aml-scikit-learn@latest\",\n",
+    "    compute=\"cpu-cluster\",\n",
+    "    experiment_name=\"train_model_credit_default_prediction\",\n",
+    "    display_name=\"credit_default_prediction\"   \n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Submit the job \n",
+    "\n",
+    "It's now time to submit the job to run in AzureML. This time you'll use `create_or_update`  on `ml_client.jobs`.\n",
+    "\n",
+    "Once completed, the job will register a model in your workspace as a result of training. You can view the job in AzureML stidio by clicking on the link in the output of the next cell."
```

Comment:
I think we need to beef up this section.  How long do we anticipate till training is complete?  They need to know that even though the cell ran, they aren't yet ready to run the next one till training is finished.  Maybe something like: Click on the link in the output of the next cell.  Wait until the status of the job is complete before returning to this notebook to continue. And give some estimate of how long that might be.  

(Also, looks like they might have to refresh the page to actually see the completed status. :-( we should mention that as well.)